### ISSUE TITLE
Initial theme selection dialog does not save choice

### ISSUE THREAD

**Issue by Tom1223** [Epix37/Hearthstone-Deck-Tracker#2288]
Choosing classic as preferred theme (ragnaros pic) it changes to one of the new ones after rebooting hdt. How do I get classic to stick?

**Comment by andburn** [comment]
go to `options > appearance > card theme` and select *Classic*

**Comment by Tom1223** [comment]
many thanks : )

**Comment by TemporalOnline** [comment]
I already tried that, it always revert to the squared one :(

**Comment by Gr3yz** [comment]
Is there a way to go back to the old font?  I don't like the new comic sans-esque one.

**Comment by Mistrelvous** [comment]
Thanks anburn, was freaking out about possibly losing the great-looking classic gems. :)

**Comment by Epix37** [comment]
Fixed the issue where the choice from the initial dialog is not saved properly.

@TemporalOnline do you have that problem with other settings as well? It might be that your config is just not saving for whatever reason. The theme saved just fine for me every time.

@Gr3yz No, sorry. Also [ChunkFive](https://www.fontsquirrel.com/fonts/chunkfive) is not even remotely close to comic sans.

**Comment by TemporalOnline** [comment]
@Epix37 I think my problem was that I changed my HDT and games directory to a directory(where I installed HDT) inside my hearthstone installation, because I use the same one in various machines (and OSs, I play it on my linux machine also), but don't worry, I don't report linux problems, those I fix myself.

What I'm trying to say is that I have my settings on a non-standard (non-appdata folder) for convenience/easy moving and sometimes it seems I have to redo my setting (also, sometimes after updating and asking to show my "appdata" folder, it crashes). It stuck after I redid my setting.
